axum-macros/Cargo.toml

```diff
@@ -16,9 +16,10 @@ proc-macro = true
 
 [dependencies]
 heck = "0.4"
+itertools = "0.10.3"
```

Comment:
not sure if there are rules around adding third-party libraries, but this makes computing cross-product much easier